dolphinscheduler-ui/src/views/projects/task/components/node/format-data.ts

```diff
@@ -533,6 +540,38 @@ export function formatModel(data: ITaskData) {
     params.others = data.taskParams.sparkParameters.others
   }
 
+  if (data.taskParams?.namespace) {
```

Comment:
Lines 543-574 are redundant. You can delete it

Reply:
Sure, thank you for the nice reminder, removed.